### ISSUE TITLE
Doesn't support schemas

### ISSUE THREAD

**Issue by sbowman** [mgutz/dat#46]
When using schemas, table names are quoted incorrectly by dat in the builders.  For example:

```
db.Update("hello.world").Set("name", "John Doe").Where("id = $1", 23).Exec()
```

Generates a query that looks like:

```
update "hello.world" set "name" = 'John Doe' where "id" = 23
```

It should be:

```
update "hello"."world" set "name" = 'John Doe' where "id" = 23
```

**Comment by sbowman** [comment]
InsertInto seems to be ok though.

**Comment by sbowman** [comment]
As is Select...From

**Comment by mgutz** [comment]
will be fixed and closed as part of #28
